packages/ruleset/src/functions/index.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
   mergePatchProperties: require('./merge-patch-properties'),
   operationIdCasingConvention: require('./operationid-casing-convention'),
   operationIdNamingConvention: require('./operationid-naming-convention'),
+  operationRequestBody: require('./operation-requestbody'),
```

Comment:
I know this is picky 😅 but to be consistent with my comment from last time - it really helps me when debugging rules and digging through code to have the rule/function files match the rule ID (i.e. `no-operation-requestbody`) when possible. If it's a hassle, I don't mind leaving it but my guess is it'll be less intuitive down the road than it seems now

Reply:
actually, I had initially was going to use "ibm-operation-requestbody" as the rule name, but then changed to "ibm-no-operation-requestbody" and forgot to rename the files after that.  will fix

Reply:
done